heareval/tasks/pipeline.py

```diff
@@ -441,12 +441,20 @@ def run(self):
             )
 
             if self.data_config["embedding_type"] == "scene":
-                # For scene labeling each scene should have one
-                assert len(audiolabel_df) == len(audiodf)
```

Comment:
Honestly this is the wrong place to do this. We should be doing this as early in the pipeline as possible. Why not do this in ExtractMetadata so that the metadata format is consistent through the entire pipeline, rather than changing halfway through?

Reply:
Sure, that sounds like a good place to perform the check. Let me see if there is a logical place to insert it.

Reply:
Done -- moved up to to ExtractMetadata